src/payment/strategies/braintree/braintree-sdk-creator.ts

```diff
@@ -43,19 +54,23 @@ export default class BraintreeSDKCreator {
                 this.getClient(),
                 this._braintreeScriptLoader.loadPaypal(),
             ])
-            .then(([client, paypal]) => paypal.create({ client }));
+                .then(([client, paypal]) => paypal.create({ client }));
         }
 
         return this._paypal;
     }
 
-    getPaypalCheckout(): Promise<BraintreePaypalCheckout> {
+    getPaypalCheckout(renderButtonCallback: RenderButtons, context: CheckoutButtonStrategy): Promise<BraintreePaypalCheckout> {
         if (!this._paypalCheckout) {
             this._paypalCheckout = Promise.all([
                 this.getClient(),
                 this._braintreeScriptLoader.loadPaypalCheckout(),
             ])
-                .then(([client, paypalCheckout]) => paypalCheckout.create({ client }));
+                .then(([client, paypalCheckout]) => paypalCheckout.create({ client }, (_error: string, instance: any) =>  {
```

Comment:
It seems it is possible that `paypalCheckout.create` call can fail. We should probably do something to handle the `error` variable that's passed into this callback. Secondly, we should avoid using `any` to declare type.

Reply:
fixed